# ACR RCADS–BCA stimulus-family follow-up

## tl;dr

Neutral Angry was excluded and two distinct post-hoc families were analyzed: Neutral Happy/Sad as a Happy/Sad facial-expression subset, and Negative/Positive Valence as a separate valence-stimulus family. Splitting the families did not rescue the broad amplitude findings: corrected Pearson and mixed-model findings in both families disappeared when P27 was omitted, and no broad Spearman amplitude result survived correction. The focused Social Phobia–Neutral Sad ROT-minus-LOT association was unchanged and remained significant in the four-test Social/Panic follow-up both with and without P27. Under the broader 12-test facial lateralization screen, however, it missed correction without P27. This remains a promising but post-hoc association, not a confirmed sad-specific effect.

## Context & Methods

The analysis used raw summed BCA from six ROIs and six raw RCADS subscales. The facial-expression subset contained Neutral Happy and Neutral Sad. The valence family contained Negative Valence and Positive Valence. Neutral Angry was excluded at the user's request. All four retained conditions were completed by all 35 participants.

Within each stimulus family, the broad amplitude screen contained 72 tests: two conditions by six ROIs by six RCADS scales. Pearson correlations were BH-FDR corrected across 72 tests, with separately corrected Spearman correlations. Condition-specific random-intercept LMMs compared `BCA ~ ROI` with `BCA ~ ROI × score`, with BH correction across 12 condition-by-scale tests per family. Family-level random-intercept LMMs compared `BCA ~ Condition × ROI` with `BCA ~ Condition × ROI × score`; the six scale-level omnibus tests were Holm-corrected within each family.

Lateralization was ROT minus LOT. The broad lateralization screen contained 12 tests per family. The narrower, previously motivated Social Phobia/Panic analysis contained four tests per family and used Holm correction. Every test family was repeated with all participants and with P27 omitted as an influence sensitivity. These stimulus families were defined after inspecting earlier results and therefore remain post hoc.

### Key Assumptions

P27 omission is an influence sensitivity, not an automatic exclusion rule. Raw and T scores are not independent evidence. Group was not included as a covariate because the anxious/non-anxious split is effectively derived from RCADS anxiety scores. A finding in one stimulus family but not the other does not itself prove a difference between families.

In [1]:
from pathlib import Path
import json
import subprocess
import sys

import pandas as pd
from IPython.display import display

SCRIPT_DIR = Path(r"C:\Users\zcm58\PyCharmProjects\FPVS-Toolbox-Repo\outputs\rcads-bca-complete-followup-20260817")
RESULTS_DIR = SCRIPT_DIR / "stimulus_family_followup_results"
SCRIPT = SCRIPT_DIR / "run_stimulus_family_followup.py"
completed = subprocess.run(
    [sys.executable, str(SCRIPT)],
    cwd=SCRIPT_DIR,
    check=True,
    capture_output=True,
    text=True,
)
print("Stimulus-family analysis regenerated successfully.")

Stimulus-family analysis regenerated successfully.


## Data

In [2]:
summary = json.loads((RESULTS_DIR / "analysis_summary.json").read_text())
audit = pd.DataFrame(
    {
        "Analysis item": [
            "Participants",
            "Participants without P27",
            "Conditions per family",
            "Broad amplitude tests per family",
            "Condition-specific LMM tests per family",
            "Broad lateralization tests per family",
            "Focused lateralization tests per family",
        ],
        "Value": [35, 34, 2, 72, 12, 12, 4],
    }
)
display(audit)
display(pd.DataFrame(
    [(family, ", ".join(conditions)) for family, conditions in summary["stimulus_families"].items()],
    columns=["Stimulus family", "Conditions"],
))

,Analysis item,Value
0,Participants,35
1,Participants without P27,34
2,Conditions per family,2
3,Broad amplitude tests per family,72
4,Condition-specific LMM tests per family,12
5,Broad lateralization tests per family,12
6,Focused lateralization tests per family,4


,Stimulus family,Conditions
0,Facial expression,"Neutral Happy, Neutral Sad"
1,Valence,"Negative Valence, Positive Valence"


## Results

In [3]:
broad = pd.read_csv(RESULTS_DIR / "family_amplitude_correlations.csv")
count_rows = []
for (family, variant), cell in broad.groupby(["Stimulus Family", "Variant"], sort=False):
    count_rows.append(
        {
            "Stimulus family": family,
            "Variant": variant,
            "Pearson BH discoveries": int((cell["Pearson q (BH-FDR)"] < .05).sum()),
            "Spearman BH discoveries": int((cell["Spearman q (BH-FDR)"] < .05).sum()),
        }
    )
display(pd.DataFrame(count_rows))

amplitude_hits = broad.loc[
    broad["Pearson q (BH-FDR)"].lt(.05),
    [
        "Stimulus Family", "Variant", "Condition", "ROI", "RCADS Scale",
        "Pearson r", "Pearson p", "Pearson q (BH-FDR)",
        "Spearman rho", "Spearman p",
    ],
]
display(amplitude_hits.round(4))

,Stimulus family,Variant,Pearson BH discoveries,Spearman BH discoveries
0,Facial expression,All 35 participants,2,0
1,Facial expression,P27 omitted,0,0
2,Valence,All 35 participants,4,0
3,Valence,P27 omitted,0,0


,Stimulus Family,Variant,Condition,ROI,RCADS Scale,Pearson r,Pearson p,Pearson q (BH-FDR),Spearman rho,Spearman p
9,Facial expression,All 35 participants,Neutral Happy,ROT,Separation Anxiety,0.6088,0.0001,0.0063,-0.1324,0.4482
45,Facial expression,All 35 participants,Neutral Sad,ROT,Separation Anxiety,0.5926,0.0002,0.0063,0.1054,0.5468
153,Valence,All 35 participants,Positive Valence,ROT,Separation Anxiety,0.6506,0.0000,0.0008,0.1631,0.3493
154,Valence,All 35 participants,Positive Valence,ROT,Generalized Anxiety,0.5120,0.0017,0.0300,0.2332,0.1775
189,Valence,All 35 participants,Negative Valence,ROT,Separation Anxiety,-0.6878,0.0000,0.0004,-0.3526,0.0378
190,Valence,All 35 participants,Negative Valence,ROT,Generalized Anxiety,-0.5343,0.0009,0.0227,-0.2362,0.1719


With P27 included, two facial-expression and four valence Pearson amplitude correlations survived BH-FDR correction. All involved ROT and predominantly Separation Anxiety or Generalized Anxiety. None had corrected Spearman support. After omitting P27, neither stimulus family contained a corrected Pearson or Spearman amplitude association.

In [4]:
focused_amplitude = pd.read_csv(
    RESULTS_DIR / "focused_family_amplitude_correlations.csv"
)
focused_amp_near = focused_amplitude.loc[
    focused_amplitude["Spearman q (BH-FDR)"].lt(.06),
    [
        "Stimulus Family", "Variant", "Condition", "ROI", "RCADS Scale",
        "Pearson r", "Pearson p", "Spearman rho", "Spearman p",
        "Spearman q (BH-FDR)",
    ],
]
display(focused_amp_near.round(4))

,Stimulus Family,Variant,Condition,ROI,RCADS Scale,Pearson r,Pearson p,Spearman rho,Spearman p,Spearman q (BH-FDR)
11,Facial expression,All 35 participants,Neutral Happy,CP,Panic Disorder,0.2220,0.2000,0.5034,0.0021,0.0493
35,Facial expression,P27 omitted,Neutral Happy,CP,Panic Disorder,0.2191,0.2131,0.5056,0.0023,0.0548


In the narrower 24-test Social/Panic amplitude family, no Pearson association survived correction in either stimulus class. Neutral Happy CP–Panic showed a rank-only trend with P27, Spearman `rho=.503`, BH `q=.049`, and remained nearly identical without P27, `rho=.506`, but narrowly missed correction, `q=.055`. This is a borderline monotonic pattern rather than established linear evidence. No focused valence amplitude result survived correction.

In [5]:
condition_models = pd.read_csv(RESULTS_DIR / "family_condition_specific_lmms.csv")
family_models = pd.read_csv(RESULTS_DIR / "family_amplitude_lmms.csv")
lmm_counts = (
    condition_models.assign(
        significant=condition_models["LMM Omnibus q (BH-FDR)"].lt(.05)
    )
    .groupby(["Stimulus Family", "Variant"], sort=False)["significant"]
    .sum()
    .reset_index(name="Significant condition-specific LMMs (of 12)")
)
display(lmm_counts)
display(
    family_models.loc[
        family_models["Effect"].eq("Any score-related amplitude association"),
        [
            "Stimulus Family", "Variant", "RCADS Scale",
            "Likelihood-Ratio Chi-Square", "Degrees of Freedom", "p",
            "Family Amplitude LMM p (Holm)", "Converged",
        ],
    ].round(4)
)

,Stimulus Family,Variant,Significant condition-specific LMMs (of 12)
0,Facial expression,All 35 participants,8
1,Facial expression,P27 omitted,0
2,Valence,All 35 participants,5
3,Valence,P27 omitted,0


,Stimulus Family,Variant,RCADS Scale,Likelihood-Ratio Chi-Square,Degrees of Freedom,p,Family Amplitude LMM p (Holm),Converged
0,Facial expression,All 35 participants,Social Phobia,31.4772,12,0.0017,0.0050,True
3,Facial expression,All 35 participants,Panic Disorder,20.3642,12,0.0605,0.1210,True
6,Facial expression,All 35 participants,Major Depression,33.6131,12,0.0008,0.0031,True
9,Facial expression,All 35 participants,Separation Anxiety,126.3793,12,0.0000,0.0000,True
12,Facial expression,All 35 participants,Generalized Anxiety,71.0319,12,0.0000,0.0000,True
15,Facial expression,All 35 participants,Obsessive Compulsive,4.4292,12,0.9744,0.9744,True
18,Facial expression,P27 omitted,Social Phobia,8.4320,12,0.7505,1.0000,True
21,Facial expression,P27 omitted,Panic Disorder,16.5984,12,0.1653,0.9920,True
24,Facial expression,P27 omitted,Major Depression,2.8589,12,0.9965,1.0000,True
27,Facial expression,P27 omitted,Separation Anxiety,6.8539,12,0.8671,1.0000,True


The condition-specific LMM screen found eight corrected facial-expression and five corrected valence results with P27, but zero in either family without P27. Family-level omnibus LMMs showed the same pattern. The all-participant Social Phobia, Major Depression, Separation Anxiety, and Generalized Anxiety models were significant in the facial subset; Major Depression, Separation Anxiety, and Generalized Anxiety were significant in the valence family. Every one became nonsignificant when P27 was omitted. Thus, separating the tasks does not make the absolute-amplitude findings robust.

In [6]:
broad_lat = pd.read_csv(RESULTS_DIR / "family_lateralization_correlations.csv")
focused_lat = pd.read_csv(RESULTS_DIR / "focused_family_lateralization_correlations.csv")
sad_social_broad = broad_lat.loc[
    broad_lat["Stimulus Family"].eq("Facial expression")
    & broad_lat["Condition"].eq("Neutral Sad")
    & broad_lat["RCADS Scale"].eq("Social Phobia"),
    [
        "Variant", "N", "Pearson r", "Pearson p",
        "Pearson q (BH-FDR)", "Spearman rho", "Spearman p",
        "Spearman q (BH-FDR)",
    ],
]
display(sad_social_broad.round(4))

focused_summary = focused_lat[
    [
        "Stimulus Family", "Variant", "Condition", "RCADS Scale", "N",
        "Pearson r", "Pearson p (Holm)", "Spearman rho",
        "Spearman p (Holm)",
    ]
]
display(focused_summary.round(4))

,Variant,N,Pearson r,Pearson p,Pearson q (BH-FDR),Spearman rho,Spearman p,Spearman q (BH-FDR)
6,All 35 participants,35,0.4230,0.0113,0.0272,0.5044,0.0020,0.0240
18,P27 omitted,34,0.4533,0.0071,0.0852,0.4656,0.0055,0.0662


,Stimulus Family,Variant,Condition,RCADS Scale,N,Pearson r,Pearson p (Holm),Spearman rho,Spearman p (Holm)
0,Facial expression,All 35 participants,Neutral Happy,Social Phobia,35,0.2707,0.3208,0.0303,1.0000
1,Facial expression,All 35 participants,Neutral Happy,Panic Disorder,35,0.1773,0.3208,-0.1007,1.0000
2,Facial expression,All 35 participants,Neutral Sad,Social Phobia,35,0.4230,0.0454,0.5044,0.0080
3,Facial expression,All 35 participants,Neutral Sad,Panic Disorder,35,0.2772,0.3208,0.2036,0.7223
4,Facial expression,P27 omitted,Neutral Happy,Social Phobia,34,-0.0635,0.7211,-0.0478,0.9406
5,Facial expression,P27 omitted,Neutral Happy,Panic Disorder,34,-0.2851,0.3066,-0.1781,0.9406
6,Facial expression,P27 omitted,Neutral Sad,Social Phobia,34,0.4533,0.0284,0.4656,0.0221
7,Facial expression,P27 omitted,Neutral Sad,Panic Disorder,34,0.1741,0.6495,0.1480,0.9406
8,Valence,All 35 participants,Negative Valence,Social Phobia,35,-0.2627,0.3507,-0.1062,1.0000
9,Valence,All 35 participants,Negative Valence,Panic Disorder,35,-0.2312,0.3507,-0.1006,1.0000


In [7]:
focused_models = pd.read_csv(RESULTS_DIR / "focused_family_lateralization_lmms.csv")
display(
    focused_models[
        [
            "Stimulus Family", "Variant", "RCADS Scale", "Effect",
            "Likelihood-Ratio Chi-Square", "p",
            "Focused Lateralization LMM p (Holm)", "Converged",
        ]
    ].round(4)
)

,Stimulus Family,Variant,RCADS Scale,Effect,Likelihood-Ratio Chi-Square,p,Focused Lateralization LMM p (Holm),Converged
0,Facial expression,All 35 participants,Social Phobia,Average score slope across conditions,4.0733,0.0436,0.1743,True
1,Facial expression,All 35 participants,Social Phobia,Condition x score interaction,0.1200,0.7290,1.0000,True
2,Facial expression,All 35 participants,Panic Disorder,Average score slope across conditions,1.6908,0.1935,0.5805,True
3,Facial expression,All 35 participants,Panic Disorder,Condition x score interaction,0.0512,0.8210,1.0000,True
4,Facial expression,P27 omitted,Social Phobia,Average score slope across conditions,2.6704,0.1022,0.2045,True
5,Facial expression,P27 omitted,Social Phobia,Condition x score interaction,6.2830,0.0122,0.0488,True
6,Facial expression,P27 omitted,Panic Disorder,Average score slope across conditions,0.1118,0.7381,0.7381,True
7,Facial expression,P27 omitted,Panic Disorder,Condition x score interaction,4.3519,0.0370,0.1109,True
8,Valence,All 35 participants,Social Phobia,Average score slope across conditions,1.1272,0.2884,0.5768,True
9,Valence,All 35 participants,Social Phobia,Condition x score interaction,6.2990,0.0121,0.0483,True


In the four-test Social/Panic facial follow-up, Neutral Sad Social Phobia lateralization remained significant with all participants, Pearson `r=.423`, Holm `p=.045`, and Spearman `rho=.504`, Holm `p=.008`, and without P27, `r=.453`, Holm `p=.028`, and `rho=.466`, Holm `p=.022`. There was no corresponding focused valence result. When all six RCADS scales were included in the broader 12-test facial lateralization family, the same Neutral Sad coefficient survived BH correction with P27 but narrowly missed without P27, Pearson `q=.085` and Spearman `q=.066`. The effect size is stable; its multiplicity classification depends on the scientifically justified scope of the post-hoc family.

The direct Social Phobia Sad-versus-Happy interaction was nonsignificant with P27 and significant only without P27. The valence Social Phobia condition interaction showed the opposite sensitivity, surviving correction only with P27. Neither interaction is stable across the influence analysis, so condition specificity should not be claimed.

## Takeaways

Separating Happy/Sad facial-expression processing from the valence task is the more defensible analysis structure, but it does not materially strengthen the broad RCADS–BCA findings. Absolute-amplitude correlations and LMM effects in both families remain P27-dependent. The only association whose magnitude and direction remain stable after P27 omission is Social Phobia with greater ROT-minus-LOT lateralization during Neutral Sad. It survives the pre-existing four-test Social/Panic follow-up but not the broader 12-test facial family without P27. The honest conclusion is therefore a moderate, hypothesis-generating Neutral Sad lateralization association that deserves preregistered replication, not evidence of a general amplitude relationship, a confirmed sad-specific mechanism, or a biomarker.